src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1274,16 +1274,16 @@
     <Kind Name="AnonymousObjectMemberDeclarator"/>
     <Field Name="NameEquals" Type="NameEqualsSyntax" Optional="true">
       <PropertyComment>
-        <summary>NameEqualsSyntax representing the optional name of the property being initialized.</summary>
+        <summary>NameEqualsSyntax representing the optional name of the member being initialized.</summary>
```

Comment:
Given that the type name was changed from `AnonymousObjectPropertyInitializerSyntax` to `AnonymousObjectMemberDeclaratorSyntax` I figured that the comment should have also been updated